File: bonheur/submissions/time_limit_exceeded/naive.py
# TLE expected

n, k = map(int, input().split())
a = list(map(int, input().split()))


for i in range(n):
    for j in range(i+1, n):
        if a[i] + a[j] == k:
            print("yes")
            exit()

print("no")

File: bonheur/submissions/time_limit_exceeded/naive.cpp
#include <iostream>
#include <vector>

#define int long long
using namespace std;

signed main() {
    int n, k; cin >> n >> k;
    vector<int> a(n);
    for (int i = 0; i < n; i++) {
        cin >> a[i];
    }
    for (int i = 0; i < n; i++) {
        for (int j = i + 1; j < n; j++) {
            if (a[i] + a[j] == k) {
                cout << "yes" << endl;
                exit(0);
            }
        }
    }
    cout << "no" << endl;

}
